--- drivers/video/Makefile.orig	2009-10-23 03:27:18.000000000 +0900
+++ drivers/video/Makefile	2009-12-17 21:55:42.000000000 +0900
@@ -125,6 +125,7 @@
 obj-$(CONFIG_XEN_FBDEV_FRONTEND)  += xen-fbfront.o
 obj-$(CONFIG_FB_CARMINE)          += carminefb.o
 obj-$(CONFIG_FB_MB862XX)	  += mb862xx/
+obj-$(CONFIG_FB_UDL)              += udlfb/
 
 # Platform or fallback drivers go here
 obj-$(CONFIG_FB_UVESA)            += uvesafb.o
